css/amp.css

```diff
@@ -155,7 +155,8 @@ i-amphtml-sizer {
   display: block;
   width: 1px;          /* These lines are a work around to this issue in iOS: */
   min-width: 100%;     /* https://bugs.webkit.org/show_bug.cgi?id=155198      */
-  height: 100%;
+  height: 1px;
```

Comment:
`+` @dvoytenko to confirm. Yuxuan ran into the same webkit bug with iframe height, the trick used for width fixed it for height as well.